smart-contracts/scripts/deploy-governor.js

```diff
@@ -0,0 +1,108 @@
+const { ethers, upgrades } = require('hardhat')
+const OZ_SDK_EXPORT = require('../openzeppelin-cli-export.json')
+
+const { getNetworkName } = require('../helpers/network')
+const { getDeployment, addDeployment } = require('../helpers/deployments')
+
+const ZERO_ADDRESS = web3.utils.padLeft(0, 40)
+const TIMELOCK_ADMIN_ROLE =
+  '0x5f58e3a2316349923ce3780f8d587db2d72378aed66a8261c916544fa6846ca5'
```

Comment:
Hey @clemsos do u mind claryfing theses addresses please?

Reply:
sure. added the keccak from [oz contracts](https://github.com/OpenZeppelin/openzeppelin-contracts/blob/fb36326a58f1bf05896014d540ba7190886e5673/contracts/governance/TimelockController.sol#L23)